src/VisualStudio/CSharp/Impl/ObjectBrowser/CSharpLibraryServiceFactory.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Composition;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Host;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.VisualStudio.LanguageServices.Implementation.Library;
+
+namespace Microsoft.VisualStudio.LanguageServices.CSharp.ObjectBrowser
+{
+    [ExportLanguageServiceFactory(typeof(ILibraryService), LanguageNames.CSharp), Shared]
```

Comment:
I don't think you need to export a factory if you don't need the `HostLanguageServices`, just an `ExportLanguageService` on the other class should be sufficient.